orders.go

```diff
@@ -293,6 +289,22 @@ type OrderDTO struct {
 	UpdatedAt                *time.Time              `json:"updated_at,omitempty"`
 }
 
+type CreateOrderDTO struct {
+	ClosedAt       *time.Time   `json:"closed_at,omitempty"`
+	CreatedAt      *time.Time   `json:"created_at,omitempty"`
+	Email          string       `json:"email,omitempty"`
+	ID             int64        `json:"id,omitempty"`
+	LineItems      LineItemDTOs `json:"line_items,omitempty"`
+	Name           string       `json:"name,omitempty"`
+	OrderNumber    int          `json:"order_number,omitempty"`
+	ProcessedAt    *time.Time   `json:"processed_at,omitempty"`
+	TotalPrice     string       `json:"total_price,omitempty"`
```

Comment:
Is this the minimum number of values we need before shopify complains? I don't remember  seeing values like TotalTax & TotalTaxSet, if they're not needed can they be removed?